<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GUIClass</class>
 <widget class="QMainWindow" name="GUIClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GUI</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QListWidget" name="feedListWidget"/>
      </item>
      <item>
       <widget class="QListWidget" name="postsListWidget"/>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QListWidget" name="topicListWidget"/>
    </item>
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QPushButton" name="addButton">
        <property name="text">
         <string>ADD</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="idLineEdit"/>
      </item>
      <item row="0" column="2" colspan="2">
       <widget class="QLineEdit" name="textLineEdit"/>
      </item>
      <item row="1" column="1" colspan="2">
       <widget class="QLineEdit" name="timeLineEdit"/>
      </item>
      <item row="1" column="3">
       <widget class="QLineEdit" name="userLineEdit"/>
      </item>
      <item row="2" column="0">
       <widget class="QPushButton" name="updateButton">
        <property name="text">
         <string>UPDATE</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1" colspan="2">
       <widget class="QLineEdit" name="oldLineEdit"/>
      </item>
      <item row="2" column="3">
       <widget class="QLineEdit" name="newLineEdit"/>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="subscribeButton">
      <property name="text">
       <string>Subscribe</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="subscribeLineEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="GUI.qrc"/>
 </resources>
 <connections/>
</ui>
